### ISSUE TITLE
Is not operator supported?

### ISSUE THREAD

**Issue by xilh** [cparse/cparse#86]
Hello, I want to use experssion like "!connected". Is there a way to do this? Thank you.

**Comment by VinGarcia** [comment]
Yes it's possible, you could add it by following the Wiki instructions here:

https://github.com/cparse/cparse/wiki/Defining-New-Operations#declaring-unary-operations

But since this is a very sane thing to be on the builtin operations, I just added it myself:

11ff91ca0c694ee5dc0f28252eef9c169c848c7c

So you just have to pull the most recent version of this repo.

Also sorry I took so long to see your question. If everything is solved we can close this issue.
